site/zenodo_rdm/openaire/serializers/schema.py

```diff
@@ -196,10 +196,15 @@ def _reverse_funder_acronym(funder_ror):
                 funder_acronym = _reverse_funder_acronym(funder_ror)
                 award_program = award.get("program", "")
                 award_number = award.get("number", "")
+                award_title = award.get("title", {}).get("en")
+                award_acronym = award.get("acronym", "")
                 if funder_acronym and award_program and award_number:
-                    links.append(
-                        f"info:eu-repo/grantAgreement/{funder_acronym}/{award_program}/{award_number}"
-                    )
+                    b_link = f"info:eu-repo/grantAgreement/{funder_acronym}/{award_program}/{award_number}"
+                    if award_title:
+                        b_link += f"/{award_title}"
+                    if award_acronym:
+                        b_link += f"/{award_acronym}"
+                    links.append(b_link)
```

Comment:
major: if `award_title` or `award_acronym` is missing, then we still include them in the final string as empty values. i.e. if both were empty we would just have `info:eu-repo/grantAgreement/EC/H2020/12345//`. See for reference the [logic in legacy](https://github.com/zenodo/zenodo/blob/482ee72ad501cbbd7f8ce8df9b393c130d1970f7/zenodo/modules/openaire/schema.py#L132-L135).